src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -263,7 +289,12 @@ public Conference createConference(
 
         do
         {
-            String id = generateConferenceID();
+            String id;
+            if(useNameForConferenceId && !StringUtils.isNullOrEmpty(name)) {
+                id = name;
+            } else {
+                id = generateConferenceID();
+            }
```

Comment:
sure I can fix the braces add a check for the id, do you have a preference on the exception that should be thrown?